src/main/java/ai/lucidtech/las/sdk/Client.java

```diff
@@ -45,315 +44,1079 @@ public Client(Credentials credentials) {
      * @throws MissingAccessTokenException Raised if access token cannot be obtained
      */
-    public JSONObject createPrediction(
-        String documentId,
-        String modelName,
-        Map<String, Object> options
+    public JSONObject deleteUser(String userId) throws IOException, APIException, MissingAccessTokenException {
+        HttpUriRequest request = this.createAuthorizedRequest("DELETE", "/users/" + userId);
+        String response = this.executeRequest(request);
+        return new JSONObject(response);
+    }
+
+    /**
+     * Creates a new workflow, calls the POST /workflows endpoint.
+     * Check out Lucidtech's tutorials for more info on how to create a workflow.
+     *
+     * @see CreateWorkflowOptions
+     * @param specification Specification of the workflow,
+     * currently supporting ASL: https://states-language.net/spec.html
+     * @param options Additional options to include in request body
+     * @return Workflow response from API
+     * @throws IOException General IOException
+     * @throws APIException Raised when API returns an erroneous status code
+     * @throws MissingAccessTokenException Raised if access token cannot be obtained
+     */
+    public JSONObject createWorkflow(
+        JSONObject specification,
+        CreateWorkflowOptions options
     ) throws IOException, APIException, MissingAccessTokenException {
-        JSONObject jsonBody = new JSONObject();
-        jsonBody.put("documentId", documentId);
-        jsonBody.put("modelName", modelName);
+        JSONObject body = new JSONObject();
+        body.put("specification", specification);
 
-        for (Map.Entry<String, Object> option: options.entrySet()) {
-            jsonBody.put(option.getKey(), option.getValue());
+        if (options != null) {
+            body = options.addOptions(body);
         }
 
-        HttpUriRequest request = this.createAuthorizedRequest("POST", "/predictions", jsonBody);
+        HttpUriRequest request = this.createAuthorizedRequest("POST", "/workflows", body);
         String jsonResponse = this.executeRequest(request);
         return new JSONObject(jsonResponse);
     }
 
     /**
-     * Create a prediction on a document <i>documentPath</i> by path using model <i>modelName</i>.
-     * This method takes care of creating and uploading a document as well as running inference using
-     * model to create prediction on the document.
+     * Creates a new workflow, calls the POST /workflows endpoint.
+     * Check out Lucidtech's tutorials for more info on how to create a workflow.
      *
-     * @param documentPath Path to document to run inference on
-     * @param modelName The name of the model to use for inference
-     * @param consentId An identifier to mark the owner of the document handle
-     * @return Prediction on document
+     * @param specification Specification of the workflow,
+     * currently supporting ASL: https://states-language.net/spec.html
```

Comment:
A link here to supported features in ASL would be nice

Reply:
Does that exist?

Reply:
I think you are the one who wrote this https://docs.lucidtech.ai/getting-started/tutorials/tutorial_custom_predict_and_approve#creating-the-workflow